packages/contracts-ops/contracts/scripts/UpgradeCallBatches.s.sol

```diff
@@ -78,3 +48,32 @@ contract UpgradeCallBatches is Script, Config, CallBatch {
         }
     }
 }
+
```

Comment:
pushUpgrade needs a check that the beacon is not already set to the correct address

Reply:
33085aa6

Reply:
https://github.com/nomad-xyz/monorepo/pull/511/commits/33085aa6cedee3a5b4cbfcbc1d71c89bc58fffc1